homeassistant/components/discovery/__init__.py

```diff
@@ -89,6 +89,7 @@
     "homekit",
     "ikea_tradfri",
     "philips_hue",
+    SERVICE_SAMSUNG_PRINTER,
```

Comment:
You can remove it from here. The `migrated` ones are if they have step_discovery, but we won't add those anymore in favor of the manifest based discovery.

Reply:
If it's not listed here, don't we get a `Unknown service discovered: ...` for it? IIRC that's why I added it here, as there seemed to be no other mechanism for preventing it -- the message doesn't seem like something we want to be issuing when it actually is a known and properly handled case.

Reply:
Nah, I misremembered; the message is `Found new service: samsung_printer ...` which is harmless enough, so removed this entry.